apps/token-wrapper/app/.babelrc

```diff
@@ -1,21 +1,31 @@
+ 
 {
   "presets": [
     [
       "@babel/preset-env",
       {
         "modules": false,
+        "targets": {
```

Comment:
I forget exactly, but is the browserlist in the package.json enough or do we need to specify `targets` here as well for `@babel/present-env`?